gcn/notices/core/Localization.schema.json

```diff
@@ -34,6 +30,46 @@
       "description": "Contains true when the systematic error is included and false when the systematic error is not included",
       "default": false
     },
+    "position_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis [deg]"
+    },
+    "galactic_l": {
+      "type": ["number", "null"],
+      "description": "Galactic longitude [deg]"
+    },
+    "galactic_b": {
+      "type": ["number", "null"],
+      "description": "Galactic latitude [deg]"
+    },
+    "galactic_semimajor_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis in galactic coordinates [deg]"
+    },
+    "instrument_phi": {
+      "type": ["number", "null"],
+      "description": "Instrument phi [deg]"
+    },
+    "instrument_theta": {
+      "type": ["number", "null"],
+      "description": "Instrument theta [deg]"
+    },
+    "instrument_semimajor_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis in instrument coordinates [deg]"
+    },
+    "spacecraft_phi": {
+      "type": ["number", "null"],
+      "description": "Spacecraft phi [deg]"
+    },
+    "spacecraft_theta": {
+      "type": ["number", "null"],
+      "description": "Spacecraft theta [deg]"
+    },
+    "spacecraft_semimajor_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis in Spacecraft coordinates [deg]"
+    },
```

Comment:
I am not sure that these fields belong in the location record, which describes the location of the burst, not the location of the spacecraft.

Reply:
These are burst coordinates in the frames of the instrument and the spacecraft respectively. The location of the spacecraft with respect to Earth is in the GeoLoc scheata, which I put in a separate pull.

Reply:
please remove the spacecraft phi, theta, semi_major.  Phi and Theta seem to have different definitions for different instruments.  These should be part of the instrument specific schema (e.g. glowbug), with a more detailed description of what the angle is and how it is referenced.